static/css/components/work.less

```
@@ -305,7 +306,7 @@ div.editionTools {
 .edition-omniline {
   display: flex;
   flex-direction: column;
-  padding: 10px;
+  padding: 20px;
```

```suggestion
  padding: 10px;
```
This change is not needed.